AutomatedTesting/Gem/PythonTests/Physics/tests/tick/Tick_CharacterGameplayComponentMotionIsSmooth.py

```diff
@@ -0,0 +1,97 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Test Case Title : Verify that an entity with a character gameplay component moves smoothly.
+"""
+
+
+# fmt: off
+class Tests():
+    create_entity              = ("Created test entity",                          "Failed to create test entity")
+    character_controller_added = ("Added PhysX Character Controller component",   "Failed to add PhysX Character Controller component")
+    character_gameplay_added   = ("Added PhysX Character Gameplay component",     "Failed to add PhysX Character Gameplay component")
+    character_motion_smooth    = ("Character motion passed smoothness threshold", "Failed to meet smoothness threshold for character motion")
+# fmt: on
+
+
+def Tick_CharacterGameplayComponentMotionIsSmooth():
+    """
+    Summary:
+     Create entity with PhysX Character Controller and PhysX Character Gameplay components.
+     Verify that the motion of the character controller under gravity is smooth.
+
+    Expected Behavior:
+     1) The motion of the character controller under gravity is a smooth curve, rather than an erratic/jittery movement.
+
+    Test Steps:
+     1) Load the empty level
+     2) Create an entity
+     3) Add a PhysX Character Controller Component and PhysX Character Gameplay component
+     4) Enter game mode and collect data for the character controller's z co-ordinate and the time values for a series of frames
+     5) Check if the motion of the character controller was sufficiently smooth
+
+    :return: None
+    """
+    # imports
+    import os
+    import azlmbr.legacy.general as general
+    import azlmbr.math as math
+    from editor_python_test_tools.editor_entity_utils import EditorEntity as Entity
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import TestHelper as helper
+    from editor_python_test_tools.asset_utils import Asset
+    import numpy as np
+
+    # constants
+    COEFFICIENT_OF_DETERMINATION_THRESHOLD = 1 - 1e-4 # curves with values below this are not considered sufficiently smooth
+
+    helper.init_idle()
+    # 1) Load the empty level
+    helper.open_level("Physics", "Base")
```

Comment:
Use Base level in the root folder instead.
open_level(“”, “Base”)

Reply:
Updated.